Playground/RxSwiftExtPlayground.playground/Pages/Index.xcplaygroundpage/Contents.swift

```diff
@@ -40,6 +40,7 @@
  - [filterMap()](filterMap) operator, filters out some values and maps the rest (replaces `filter` + `map` combo)
  - [Observable.fromAsync()](fromAsync) constructor, translates an async function that returns data through a completionHandler in a function that returns data through an Observable
  - [ofType()](ofType) operator, filters the elements of an observable sequence, if that is an instance of the supplied type.
+ - [UIViewPropertyAnimator.rx.animate](animate) operator, provides a Completable that when subscribed to starts the animation and completes once the animation is ended
```

Comment:
You're right about the fact this is the first Extension we have on RxCocoa. I'm not 100% sure on how to name this thing. 

Let's change for now to this - I also removed the note about "subscribing" because I think its implicit in the fact its an Observable: 

` **UIViewPropertyAnimator** [animate](animate) operator, returns a Completable that completes as soon as the animation ends.`

eg. 

> - **UIViewPropertyAnimator** [animate](animate) operator, returns a Completable that completes as soon as the animation ends.

Reply:
done.